src/storage/kv/store.rs

```diff
@@ -601,36 +601,32 @@ mod tests {
 
     #[tokio::test]
     async fn insert_close_reopen() {
-        for _ in 0..10 {
-            // Create a temporary directory for testing
-            let temp_dir = create_temp_directory();
+        // Create a temporary directory for testing
+        let temp_dir = create_temp_directory();
+
+        // Create store options with the test directory
+        let mut opts = Options::new();
+        opts.dir = temp_dir.path().to_path_buf();
 
-            // Create store options with the test directory
-            let mut opts = Options::new();
-            opts.dir = temp_dir.path().to_path_buf();
+        let num_ops = 1000;
 
-            // Create a new store instance with VariableKey as the key type
+        for i in 1..=10 {
+            // (Re)open the store
             let store = Store::new(opts.clone()).expect("should create store");
 
-            // Insert 1000 items into the store
-            for i in 0..1000 {
-                let key = format!("key{}", i);
-                let value = format!("value{}", i);
+            // Append num_ops items to the store
+            for j in (num_ops * (i - 1))..(num_ops * i) {
```

Comment:
for visibility purposes, I would move this math to a temp var, because it's not actually relevant for the loop
```
for j in 0..num_ops {
  let val = (i - 1) * num_ops + j
  let key = format!("key{}", val);
  let value = format!("value{}", val);
  ...
}
```

Reply:
ack, done